code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -169,7 +169,7 @@
 /datum/eldritch_knowledge/summon/raw_prophet
 	name = "Raw Ritual"
 	gain_text = "Uncanny man, walks alone in the valley between the worlds , I was able to call his aid."
-	desc = "You can now summon a Raw Prophet using eyes, a left arm, right arm and a pool of blood. Raw prophets have increased seeing range, as well as Xray. But are very fragile and weak."
```

Comment:
still not quite sure why you cut the right arm outta this

Reply:
wanted to make it cheaper